crates/ln-dlc-node/src/node/mod.rs

```diff
@@ -419,11 +421,7 @@ impl Node {
             address,
         };
 
-        tracing::info!(
-            "Lightning node started with node ID {}@{}",
-            node_info.pubkey,
-            node_info.address
-        );
+        tracing::info!("Lightning node started with node ID {}", node_info);
```

Comment:
Not sure if this compiles but imho this is the preferred way to print this info.
```suggestion
        tracing::info!(node_info, "Lightning node started");
```